### DISCUSSION TITLE
The time trace measure function stops at 0 if acqTime = 0

### DISCUSSION THREAD

**Discussion by thinhbia2** [PicoQuant/snAPI#35] [General]
According to snAPI document, the measurement should run until stopMeasure() called when acqTime = 0. But it stops immediately and the console displays DEV MH_GetElapsedMeasTime: 0 0ms

Would you please help?

**Comment by tpoint75** [comment]
You are welcome. Can you provide the script and the log file located at c:\data\picoquant\logs

**Reply by thinhbia2** [reply]
Please refer to the log file for that event (acqTime = 0). I used the Demo_TimeTrace.py and just changed acqTime = 0.

[240905.log](https://github.com/user-attachments/files/16897770/240905.log)

**Reply by tpoint75** [reply]
240905_19:45:08.2983691 ERR MHlib error in MH_StartMeas devIdx:0 tAqc:1 (-60): MH_ERROR_MB_ACK_FAIL
240905_19:45:08.2986350 DEV last error:      MH_ERROR_MB_ACK_FAIL 
occurred in:  MH_StartMeas 
in mode:       2 = MODE_T2

 last MB_Ack return code: 80000032

Device state:  6 = STATE_STARTING
FPGA mode:      2 
Ext clock:          0 
Sync Divider:     1 
Binning:             0 



Lib version: 3.1  Mar 22 2024

Device serial number: 1051405
USB firmware version: 0.20
M0 gateware version:  0.9
M1 gateware version:  0.9

Windows version: 10.0.22631.2
Windows name: Windows 10 Pro

There is a hardware error. I will talk tomorrow with the hardware developer.
Please switch the MultiHarp off and on and try again.

**Reply by tpoint75** [reply]
See https://github.com/PicoQuant/snAPI/issues/36
